src/uct/ib/base/ib_md.c

```diff
@@ -103,6 +103,10 @@ static ucs_config_field_t uct_ib_md_config_table[] = {
      "Use indirect atomic\n",
      ucs_offsetof(uct_ib_md_config_t, ext.enable_indirect_atomic), UCS_CONFIG_TYPE_BOOL},
 
+    {"SUBNET_PREFIX", "",
+     "Filter IB ports by subnet_prefix",
```

Comment:
minor: maybe add `(empty by default)` to the end of the description

Reply:
and mention that empty means no filter